server/sonar-ce/src/main/java/org/sonar/ce/taskprocessor/CeWorker.java

```diff
@@ -28,5 +28,13 @@
  * {@link Callable#call()} returns a Boolean which is {@code true} when some a {@link CeTask} was processed,
  * {@code false} otherwise.
  */
-public interface CeWorkerCallable extends Callable<Boolean> {
+public interface CeWorker extends Callable<Boolean> {
+
+  /**
+   * UUID of the worker.
+   * The fact that this UUID is unique across a SQ cluster and over time is mandatory for CE nodes to work correctly.
+   *
+   * @return the uuid
```

Comment:
the change has been done in the wrong commit and the description should be kept, please fix it